### DISCUSSION TITLE
File-based resource names must contain only lowercase a-z, 0-9, or  underscore

### DISCUSSION THREAD

**Discussion by realrecordzLab** [ionic-team/capacitor#7268] [Q&A]
When I try to do the build of my capacitor android app I will always get this error.
```
Execution failed for task ':app:mergeReleaseResources'.
        > /Users/b/Desktop/myApp/android/app/src/main/res/xml/config
        2.xml: Error: ' ' is not a valid file-based resource name character:
        File-based resource names must contain only lowercase a-z, 0-9, or
        underscore
``` 
The config2.xml file is generated from capactor so I don't really know what is going wrong. How I can fix this?

**Comment by jcesarmobile** [comment]
capacitor creates a config.xml file, not a config 2.xml file, maybe you copied the file manually and your computer named it like config 2.xml because config.xml already existed? try deleting the config 2.xml file manually

**Reply by emcranny** [reply]
I have the same problem and I did not manually copy anything. I created an app using ionic app "start" for a blank React app then I did the following:
1. ionic serve
2. ionic build
3. ionic cap add ios
4. ionic cap add android
5. cap open ios
6. cap open android
7. ionic cap sync
8. cap open ios
9. cap open android

now I have a file named "config 2.xml" in app/res/xml folder
